### DISCUSSION TITLE
Update request from Peugeot

### DISCUSSION THREAD

**Discussion by jguerre** [flobz/psa_car_controller#640] [General]
I am curious about how often does Peugeot request update to the car and if it is a fixed peiod or variable? I have also received the message about 12V battery excessive demand due to continuous requets and I would like to know how often the car does update to adjust the poll rate. Thanks to anyone in advance!

**Comment by Dizzybacon** [comment]
I think it doesn't request updates in general. I think the car sends updates according to other triggers, like finishing a journey or starting charging. If you trigger an update from the app of course it will request an update from the car.
